samples/dnn/mobilenet_ssd_python.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import print_function
+import numpy as np
+import sys
+sys.path.append('/home/arrybn/build/opencv/lib')
```

Comment:
Please remove this hard-coded path too. Use `PYTHONPATH` environment variable instead to `cv2.pyd` location.